### DISCUSSION TITLE
Raid Difficulty Management

### DISCUSSION THREAD

**Discussion by dreadwraith** [ldtteam/minecolonies#8406] [Ideas]
Is there a way to scale back the difficulty of raids? I have a young cousin who enjoys building his village, but after a while the mod spawns nothing but large raiding parties, leaving him with little time to actually build. He likes that the raids exist, but in his words, "The big groups of bullies are too hard." Is there a way to reduce the size or difficulty of the raids as a whole, or possibly even scale them dynamically based on how well your village fares against the previous one? Maybe base it on total death count per raid, with higher friendly deaths causing the next one to scale back a bit, or even delay the next one somewhat to give you time to recover?

**Comment by Talyda** [comment]
They actually do already scale based on previous raids =)
They maybe don't scale as fast as you would like? But they do scale!

**Comment by ArkenorOakshadow** [comment]
I recently had to deal with a stack of about 50 attackers all in one clump. Maybe they could improve in quality a bit more, and quantity a bit less.

**Reply by Raycoms** [reply]
There is a max number of raiders we allow, but we do want to avoid scaling them up in quality even more because we want them to be "similar to guards". Where you'll also have more than 50 guards in such a large colony.

**Comment by Raycoms** [comment]
If you want them scale up less there is a difficulty setting in the config.

**Comment by Raycoms** [comment]
There is a config value to decrease their strength over all, and the dynamic adjustment already exists.

Sent from ProtonMail mobile

-------- Original Message --------
